### ISSUE TITLE
`gridsome create` producing error on Install dependencies

### ISSUE THREAD

**Issue by elliottross23** [gridsome/gridsome#357]
## Description

`npm install` produces this error:

```
npm ERR! Unexpected end of JSON input while parsing near '...:{"css-loader":"^0.23'
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/elliott/.npm/_logs/2019-04-10T01_52_01_655Z-debug.log
```

Error from debug.log:

```
2429 silly saveTree   ├── webpack-dev-middleware@3.6.2
2429 silly saveTree   ├── webpack-hot-middleware@2.24.3
2429 silly saveTree   ├── webpack@4.29.6
2429 silly saveTree   └── yaml-loader@0.5.0
2430 verbose stack SyntaxError: Unexpected end of JSON input while parsing near '...:{"css-loader":"^0.23'
2430 verbose stack     at JSON.parse (<anonymous>)
2430 verbose stack     at parseJson (/Users/elliott/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/json-parse-better-errors/index.js:7:17)
2430 verbose stack     at consumeBody.call.then.buffer (/Users/elliott/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-fetch-npm/src/body.js:96:50)
2430 verbose stack     at process._tickCallback (internal/process/next_tick.js:68:7)
2431 verbose cwd /Users/elliott/Projects/my-gridsome-site
2432 verbose Darwin 18.2.0
2433 verbose argv "/Users/elliott/.nvm/versions/node/v10.15.3/bin/node" "/Users/elliott/.nvm/versions/node/v10.15.3/bin/npm" "install"
2434 verbose node v10.15.3
2435 verbose npm  v6.9.0
2436 error Unexpected end of JSON input while parsing near '...:{"css-loader":"^0.23'
2437 verbose exit [ 1, true ]
```

### Steps to reproduce

1. Run `gridsome create my-gridsome-site` and Install dependencies step will fail
2. `cd my-gridsome-site` and run `npm install` and it will fail with the above error

### Expected result

`gridsome create` should successfully install dependencies

### Actual result

`npm install` always fails

### Environment

<pre><code>
Libs:
- @gridsome/cli version: 0.0.9
 
Tooling:
- Node version: 10.15.3
- NPM version: 6.9.0
- Platform:  Mac
</code></pre>

**Comment by elliottross23** [comment]
**More info:** I'm using nvm to manage my npm versions. I also have tried to remove @gridsome/cli globally and reinstall. Still running into the same issue.

**Comment by elliottross23** [comment]
seems to be fixed after upgrading to gridsome v0.5.8
